lib/actions/artisan.js

```diff
@@ -184,19 +186,63 @@ module.exports = {
       return;
     }
 
-    // Log
-    Logs.show('server-started', {
-      type: 'Info',
-      message: null,
-      params: {}
+    // Grab the configurations
+    const config = JSON.parse(Utils.readFileSync('./site.json', 'utf-8'));
+    const portToSet = options.port || config.server.port || env.server.defaultPort;
+
+    // If the given port is different than whats in the config, update site.json with the new port
+    if (portToSet !== config.server.port) {
+      config.server.port = portToSet;
+      fsExtra.outputFileSync(`${env.templates.config}`, JSON.stringify(config, null, 2));
```

Comment:
I did tried to use `writeFileSync` but it expects the configs in a different format. Right now, the config object have two inner objects called `db` and `server` but `writeFileSync` expects an object without these two (Their values in the main object).

For example, here's the config I can pass :

```
{
  "siteName": "a",
  "siteVersion": "1.0.0",
  "cliVersion": "0.0.1",
  "db": {
    "mysqlDbHost": "127.0.0.1",
    "mysqlDbPort": "3306",
    "mysqlDbUser": "root",
    "mysqlDbPwd": "earthsoftp",
    "mysqlDbName": "abc2",
    "mysqlDbTblPrefix": "wp_"
  },
  "server": {
    "host": "localhost",
    "port": "8081"
  }
}
```

But here's what `writeFileSync` expects :

```
{ siteName: 'a',
  mysqlDbHost: '127.0.0.1',
  mysqlDbPort: '3306',
  mysqlDbUser: 'root',
  mysqlDbPwd: 'earthsoftp',
  mysqlDbName: 'abc',
  mysqlDbTblPrefix: 'wp_',
  serverHost: 'localhost',
  serverPort: '8081',
  doInstall: false,
  cliVersion: '0.0.1',
  siteUrl: 'http://localhost:8081' }
```

Might have to convert the config before passing :smile: Let's see what I can do...